aws/rust-runtime/aws-inlineable/src/presigning.rs

```diff
@@ -186,6 +186,20 @@ pub mod request {
                 .finish()
         }
     }
+
+    impl Into<http::request::Builder> for PresignedRequest {
+        fn into(self) -> http::request::Builder {
+            let mut builder = http::request::Builder::new()
+                .uri(self.uri())
+                .method(self.method());
+
+            if let Some(headers) = builder.headers_mut() {
+                *headers = self.headers().clone();
+            }
```

Comment:
this will only be `None` if the builder is invalid, so I don't think this is the right way to go here. We should probably either implement `TryInto` or use `expect` under the pretense that `uri` and method are both valid so the builder _must_ be valid